src/data/h1/tags/weapon.yml

```diff
@@ -430,17 +455,22 @@ comments:
             Controls how long it takes for the weapon to return to starting rate
             of fire when not firing.
         - name: blurred rate of fire
-          en: ...
+          en: >
+            The rate of fire needed to switch the object to its ~blur permutation.
```

Comment:
`~blur` looks like a special keyword of sorts. I've been using backtick code tags for special names. You could also make "permutation" here a link to `[gbxmodel#permutations]`.